### PR TITLE
DATAACCES:BuyerPost Repasitories Created

### LINKED ISSUE #28: DATAACCESS: Create A Buyer Posts repository

### PR DESCRIPTION
**Pull request by Yodgorovna**
Closes #28

### CODE REVIEW

File: src/GreenSaleuz.DataAccess/Repositories/BuyerPosts/BuyerPostsRepository.cs [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+public class BuyerPostsRepository : BaseRepository, IBuyerPostRepository
+{
+    public async Task<long> CountAsync()
+    {
+        try
+        {
+            await _connection.OpenAsync();
+            string query = $"SELECT COUNT(*) FROM buyer_posts;";
+            var result = await _connection.QuerySingleAsync<long>(query);
+            return result;
```
Comment (SamandarYigitaliyev) [inline src/GreenSaleuz.DataAccess/Repositories/BuyerPosts/BuyerPostsRepository.cs:18]: 1 row before return

File: src/GreenSaleuz.DataAccess/Repositories/BuyerPosts/BuyerPostsRepository.cs [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+    public async Task<int> DeleteAsync(long Id)
+    {
+        try
+        {
+            await _connection.OpenAsync();
+            string query = $"DELETE FROM public.buyer_posts WHERE id = @ID;";
+
+            var result = await _connection.ExecuteAsync(query, new { ID = Id });
+            return result;
```
Comment (SamandarYigitaliyev) [inline src/GreenSaleuz.DataAccess/Repositories/BuyerPosts/BuyerPostsRepository.cs:63]: return dan oldin enter tashlab yozish kere

**Review by SamandarYigitaliyev — COMMENTED** [review]
codlarzni clean code ga tushirib boshqattan yuklang

**Review by SamandarYigitaliyev — CHANGES_REQUESTED** [review]
Clean code ga amal qilib boshqattan to'g'irlab yuklang
shu [qoidalar](https://github.com/hassanhabib/CSharpCodingStandard) boyicha keting

**Review by rustambekovich — APPROVED** [review]
The code is very well written, but different syntaxes are used in methods that perform the same task.
I hope you will improve in your next codes

**Pull request merged**